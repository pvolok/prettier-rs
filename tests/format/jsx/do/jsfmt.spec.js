run_spec(import.meta, ["babel"], {
  errors: {
    acorn: ["do.js"],
    espree: ["do.js"],
    meriyah: ["do.js"],
  },
});
