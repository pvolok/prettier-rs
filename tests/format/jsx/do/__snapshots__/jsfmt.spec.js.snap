// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`do.js [acorn] format 1`] = `
"Unexpected token (2:4)
  1 | <div>
> 2 |   {do {
    |    ^
  3 | 	  1
  4 |   }}
  5 | </div>"
`;

exports[`do.js [espree] format 1`] = `
"Unexpected token do (2:4)
  1 | <div>
> 2 |   {do {
    |    ^
  3 | 	  1
  4 |   }}
  5 | </div>"
`;

exports[`do.js [meriyah] format 1`] = `
"Unexpected token: 'do' (2:5)
  1 | <div>
> 2 |   {do {
    |     ^
  3 | 	  1
  4 |   }}
  5 | </div>"
`;

exports[`do.js format 1`] = `
====================================options=====================================
parsers: ["babel"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div>
  {do {
	  1
  }}
</div>

=====================================output=====================================
<div>
  {do {
    1;
  }}
</div>;

================================================================================
`;
